app/src/main/java/com/mediapicker/sample/MainActivity.kt

```diff
@@ -23,9 +23,12 @@ class MainActivity : AppCompatActivity() {
     private val REQUEST_VIDEO_CAPTURE: Int = 1000
     private var fragment: HomeFragment? = null
 
+    private lateinit var binding: ActivityMainBinding
```

Comment:
@AnkurSamaryaOlxIndonesia This will create a memory leak. Please follow one of the approach as mentioned here: https://proandroiddev.com/avoiding-memory-leaks-when-using-data-binding-and-view-binding-3b91d571c150

Reply:
done